doc/cijobs.md

```diff
@@ -0,0 +1,60 @@
+# CI jobs information
```

Comment:
Some of that is in the getstarted.md. I intend to rewrite/refactor what is in there later.